Revert/Sources/Static.swift

```diff
@@ -14,6 +14,18 @@ struct Static {
     }()
   }
 
+  struct NumberFormatter {
+    static var decimalFormatter: NSNumberFormatter = {
```

Comment:
Just go with `decimal` as a name, its already enclosed in a `NumberFormatter` namespace.

You could also considering having a common `Formatter` parent struct to `Number` and `Date` now that we have two distinct types of formatters.